modules/@apostrophecms/login/ui/apos/components/TheAposLogin.vue

```diff
@@ -257,7 +256,6 @@ export default {
         this.redirectAfterLogin();
       } catch (e) {
         this.error = e.message || 'An error occurred. Please try again.';
-        this.requirements = getRequirements();
```

Comment:
I.... think this is right, but did you test it (do after-login requirements missed on the first try still at least work with this change)?

Reply:
Yes, tested with totp on afterPasswordVerified phase